<launch>

    <arg name="project" default="collect_data"/>
    
    <!-- Parameters -->
    <rosparam file="$(find collect_data)/config/data_params.yaml" command="load" />

    <arg name="episodeIndex" default="0"/>
    <arg name="datasetDir" default="/home/agilex/data"/>
    <param name="episodeIndex" value="$(arg episodeIndex)"/>
    <param name="datasetDir" value="$(arg datasetDir)"/>

    <node pkg="$(arg project)" type="$(arg project)_dataSync" name="$(arg project)_dataSync" output="screen"/>

</launch>
